### PR TITLE
Vending machines  name case fix

### PR DESCRIPTION
**Pull request by AJCM-git**
So i didnt knew that the inventory pack of the vending machines overwrote the prototype name, my bad
- Removes vending machines name in the main prototype since it is overwrote by its inventory

### CODE REVIEW

**Comment by ComicIronic** [comment]
Do these show up correctly in the `Spawn Entities` menu?

**Comment by AJCM-git** [comment]
They appear with the ID as a name, this wasnt a good idea huh

**Comment by AJCM-git** [comment]
Okay, everything is nice and dandy now

File: Resources/Prototypes/Entities/Buildings/vending_machines.yml [hunk 1]
```diff
@@ -120,7 +120,7 @@
 - type: entity
   parent: VendingMachine
   id: VendingMachineCola
-  name: Robust Softdrinks
+  name: Robust softdrinks
```
Comment (PJB3005) [inline Resources/Prototypes/Entities/Buildings/vending_machines.yml:123]: Wouldn't softdrinks still be part of the brand name in this case? And many other ones down below.

Comment (AJCM-git) [reply Resources/Prototypes/Entities/Buildings/vending_machines.yml]: Done

**Pull request merged**